RZBluetooth/Profiles/RZBDeviceInfo.h

```diff
@@ -9,6 +9,42 @@
 #import "RZBBluetoothRepresentable.h"
 #import "RZBPeripheral.h"
 
+/// Object representing a Bluetooth System ID
+/// (See https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.system_id.xml)
+@interface RZBSystemId : NSObject
+
+/// 40-bit Manufacturer-defined identifier
+@property (assign, nonatomic) UInt64 manufacturerId;
+/// 24-bit Organizationally Unique Identifier aka Company Identifier assigned by IEEE
+@property (assign, nonatomic) UInt32 ouid;
+
+@end
+
+/// Options for RZBPnPId.vendorIdSource property
+typedef enum _RZBVendorIdSource : UInt8 {
```

Comment:
Any reason not to use the `NS_ENUM` system? I'd prefer to use it if possible. Also, I'd prefer to have the values follow the prefixed pattern -- ie: `RZBVendorIdSourceReserved`.